crates/resolver/src/config.rs

```diff
@@ -802,13 +802,13 @@ impl NameServerConfigGroup {
         nameserver_ips: impl Iterator<Item = IpAddr>,
         trust_negative_response: bool,
     ) {
-        nameserver_ips.for_each(|ip| {
+        for ip in nameserver_ips {
```

Comment:
Nit: this really belongs in the other commit.

Reply:
done